src/panels/LogPanel/ScriptLogPanel.tsx

```diff
@@ -0,0 +1,76 @@
+import { useCallback, useEffect, useState } from 'react';
+import { ActionIcon, Group, Tooltip } from '@mantine/core';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+import { FilterList } from '@/components/FilterList/FilterList';
+import { wordBeginningSubString } from '@/components/FilterList/util';
+import { ReplayIcon } from '@/icons/icons';
+
+import { ScriptLogEntry } from './ScriptLogEntry';
+
+// TODO (anden88 2025-04-17): Remove when lists from OpenSpace don't have the
+// ['1']: value, ['2']: value format
+type FileLines = Record<string, string> | undefined;
+
+export function ScriptLogPanel() {
+  const luaApi = useOpenSpaceApi();
+  const [scriptLogEntries, setScriptLogEntries] = useState<string[]>([]);
```

Comment:
Maybe we want default state to be `undefined` instead of `[]` since I guess it is theoretically possible (but probably will never happen, at least not now) with an empty scriptlog? Just seeing it could potentially show loading blocks when the log is just empty